e2e/pages/vm-sidebar.po.ts

```diff
@@ -171,4 +172,142 @@ export class VMSidebar extends CloudstackUiPage {
       by.cssContainingText('.value', expected),
     );
   }
+
+  clickChangeSO() {
+    const edit = element(by.css('cs-service-offering-details .mat-icon-button'));
+    const EC = protractor.ExpectedConditions;
+    browser.wait(EC.elementToBeClickable(edit), 2000, 'No edit Group button').then(() => {
+      edit.click();
+      this.waitDialogModal();
+    });
+  }
```

Comment:
It seems it is not necessary to wait here until the item is clickable. I think this code is superfluous. Make sure you can click on the item without additional waiting. If so, then this function will be just one line of code.